core/src/main/java/bisq/core/dao/monitoring/DaoStateMonitoringService.java

```diff
@@ -142,13 +148,13 @@ public void onDaoStateChanged(Block block) {
         long totalBurntFee = daoStateService.getTotalBurntFee();
         long compensationIssuance = daoStateService.getTotalIssuedAmount(IssuanceType.COMPENSATION);
         long reimbursementIssuance = daoStateService.getTotalIssuedAmount(IssuanceType.REIMBURSEMENT);
-        long totalConfiscatedAmount = daoStateService.getTotalAmountOfConfiscatedTxOutputs();
+        long totalInvalidAmount = daoStateService.getBurnedBsqOfAllInvalidTxs();
         // confiscated funds are still in the utxo set
         long sumUtxo = daoStateService.getUnspentTxOutputMap().values().stream().mapToLong(BaseTxOutput::getValue).sum();
-        long sumBsq = genesisTotalSupply + compensationIssuance + reimbursementIssuance - totalBurntFee;
+        long sumBsq = genesisTotalSupply + compensationIssuance + reimbursementIssuance - totalBurntFee - totalInvalidAmount;
 
         if (sumBsq != sumUtxo) {
-            throw new RuntimeException("There is a mismatch between the UTXO set and the DAO state. Please contact the Bisq devlopers.");
+            utxoMismatches.add(new UtxoMismatch(block.getHeight(), sumUtxo, sumBsq));
```

Comment:
If we get a mismatch it seems that will stay mismatched in the future as well and we'll get a new `UtxoMismatch` added at every block after the first one. Maybe this is good but could be a bit annoying if there is a popup for every new block.

Reply:
Yes. Need to improve that. Popups are handled in the UI to not repeat but still we don't want to add it at each block again. But anyway should not happen at all, is just to double check if we missed something. So I consider it more as a developer tool...

Reply:
I leave that for now but will keep it on my todo list